src/core/wee-config.c

```diff
@@ -152,6 +152,7 @@ struct t_config_option *config_look_mouse;
 struct t_config_option *config_look_mouse_timer_delay;
 struct t_config_option *config_look_nick_color_force;
 struct t_config_option *config_look_nick_color_hash;
+struct t_config_option *config_look_nick_color_hash_seed;
```

Comment:
`salt` instead of `seed` would be a more appropriate name: the value is used as a salt to a hash function, not as a seed to some PRNG.